src/master-playlist-controller.js

```diff
@@ -419,6 +419,9 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
       this.trigger('audioupdate');
       this.trigger('selectedinitialmedia');
+
+      // fired when a user selects an alternate audio stream in HLS
+      this.tech_.trigger({type: 'usage', name: 'hls-audio-change'});
```

Comment:
I think it makes sense to move this event to `videojs-contrib-hls.js` in the `audioTrackChange_` method